.buildkite/src/Jobs.dhall

```diff
@@ -0,0 +1,6 @@
+-- Add any new jobs here
+[
+  ./jobs/Sample/Spec.dhall,
+  ./jobs/Sample2/Spec.dhall
```

Comment:
Should we auto-gen this dhall file?

Reply:
Yes, but I'll add a TODO for that and we can do it later